EST.MIT.InvoiceImporter.Function/Models/Notification.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+
+namespace EST.MIT.InvoiceImporter.Function.Models;
+public enum NotificationType
```

Comment:
Probably remove this enum entirely for InvoiceImporter - It's not used, so confusing I think?

Reply:
Done